test/files/boolean_rle.json

```diff
@@ -0,0 +1,17 @@
+[
+    [1],
```

Comment:
It seems like this should be `true` not `1` for this file since the column has type `BOOLEAN`. But that is a separate issue, not a blocker for getting this fix in.